modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/autocomplete/autocomplete.js

```diff
@@ -25,6 +25,8 @@
 
 	var STR_SPACE = ' ';
 
+	var BR_TAG = 'BR';
```

Comment:
Should probably sort this above `CSS_LFR_AC_CONTENT`. The list of declarations here isn't entirely sorted, but it mostly is, so we should try to keep it that way.